Content.Server/Projectiles/Components/HitscanComponent.cs

```diff
@@ -21,22 +21,24 @@ namespace Content.Server.Projectiles.Components
     public class HitscanComponent : Component
     {
         [Dependency] private readonly IGameTiming _gameTiming = default!;
-        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
 
         public override string Name => "Hitscan";
+        public CollisionGroup CollisionMask => (CollisionGroup) _collisionMask;
 
         private TimeSpan _startTime;
         private TimeSpan _deathTime;
 
         [DataField("layers")] //todo  WithFormat.Flags<CollisionLayer>()
         private int _collisionMask = (int) CollisionGroup.Opaque;
         [DataField("damage")]
-        private float _damage = 10f;
+        public float Damage { get; set; } = 10f;
 
+        // TODO PROTOTYPE Replace this datafield variable with prototype references, once they are supported.
+        // Also, should probably a be a dictionary along with the 'damage' field.
```

Comment:
there should probably be a unified way to specify a list of damages to apply; i imagine a lot of stuff can use this like HealthChange reagenteffect and MeleeWeapon, out of scope though

Reply:
Agreed. Naming wise (damage class -> group, damage -> damageType) I think I managed to make everything consistent. However, that field is used in a lot of weapons definitions, and theres still the pinned issue asking people to not touch ranged weapons, so I left it as is.

It would also be nice if damage types & groups could be specified together. E.g., for a reagent that deals a damage type while healing a group.